summingbird-online/src/main/scala/com/twitter/summingbird/online/executor/Summer.scala

```diff
@@ -63,7 +62,7 @@ class Summer[Key, Value: Semigroup, Event, S](
   maxWaitingFutures: MaxWaitingFutures,
   maxWaitingTime: MaxFutureWaitTime,
   maxEmitPerExec: MaxEmitPerExecute,
-  includeSuccessHandler: IncludeSuccessHandler) extends AsyncBase[(Int, CMap[Key, Value]), Event, InputState[S]](
+  includeSuccessHandler: IncludeSuccessHandler) extends AsyncBase[Traversable[(Key, Value)], Event, InputState[S]](
```

Comment:
Yes, now it accepts `Iterable[(Key, Value)]`, because sometime it's aggregated and sometime it's not.